docs/toolStack.md

```diff
@@ -11,6 +11,8 @@ HubTurbo is written in Java. [ControlsFX](http://fxexperience.com/controlsfx/) e
 
 Platform-specific components are used as well: [Selenium](http://www.seleniumhq.org/) for instantiating and controlling a browser window, [JNA](https://github.com/twall/jna) to manage window focus on Windows, and [jkeymaster](https://github.com/tulskiy/jkeymaster) for a global hotkey.
 
+Due to usage of ControlsFX 8.40.10, HubTurbo requires minimum Java version of 1.8.0_60.
```

Comment:
Here we are creating another place where things can become inconsistent. Perhaps this too can be added as a comment where the min version is declared?